doc/build_apps/install_bin.rst

```diff
@@ -88,3 +88,13 @@ From Source
 -----------
 
 To build and install CCF from source, please see :doc:`/contribute/build_ccf`.
+
+In Azure
+--------
+
+You can install CCF on an Azure Virtual Machine by editing the ``install_ccf_on_azure_vm.sh`` and defining your own DNS name.
```

Comment:
```suggestion
CCF can be installed on an Azure Virtual Machine by setting the resource group, VM and DNS names in the :ccf_repo:`getting_started/azure_vm/install_ccf_on_azure_vm.sh` script and running it.
```

FYI, the we use `ccf_repo` directive to link to the main repository so that links are versioned with the documentation.